rcl/include/rcl/time.h

```diff
@@ -272,20 +272,20 @@ rcl_ret_t
 rcl_difference_times(
   rcl_time_point_t * start, rcl_time_point_t * finish, rcl_duration_t * delta);
 
-/// Fill the time point with the current value of the associated clock.
+/// Fill the time point value with the current value of the associated clock.
 /**
- * This function will populate the data of the time_point object with the
+ * This function will populate the data of the time_point_value object with the
  * current value from it's associated time abstraction.
  * \param[in] clock The time source from which to set the value.
- * \param[out] time_point The time_point to populate.
+ * \param[out] time_point_value The time_point value to populate.
  * \return `RCL_RET_OK` if the last call time was retrieved successfully, or
  * \return `RCL_RET_INVALID_ARGUMENT` if any arguments are invalid, or
  * \return `RCL_RET_ERROR` an unspecified error occur.
  */
 RCL_PUBLIC
 RCL_WARN_UNUSED
 rcl_ret_t
-rcl_clock_get_now(rcl_clock_t * clock, rcl_time_point_t * time_point);
+rcl_clock_get_now(rcl_clock_t * clock, rcl_time_point_value_t * time_point_value);
```

Comment:
new functionality in rclpy is relying on the existing signature: https://github.com/ros2/rclpy/pull/209/files#diff-80dda03110b5be606b823d6b0558b5c2R3218 (CI wouldn't have picked up on that because the PR was merged after it). Sorry about that, I didn't realise that merge would cause conflict here (thought we were adding a new signature rather than modifying the existing one). I would make a PR for it but https://github.com/ros2/rclpy/pull/211 is already connected and needs to be rebased first

Reply:
Thanks for pointing it out - no problem. I rebased the `rclpy` PR and updated it in https://github.com/ros2/rclpy/pull/211/commits/f938bac8f72d7ff0af79c0ba04abfd149ce68be6 to use the modified API.